test/unittests/commonMocks.h

```diff
@@ -220,5 +224,28 @@ MATCHER_P(MatchNcar, expected, "") {
 
 }
 
+
+
+/* ****************************************************************************
+*
+* MatchHttpInfo -
+*
+* NOTE
+*   HttpInfo is a class, but after fighting with operator== quite some time I (kz) gave up
+*   and tried this way instead.
+*/
+MATCHER_P(MatchHttpInfo, expected, "")
```

Comment:
Maybe this needs to be extended to compare all the possible fields in httpInfo class?